### ISSUE TITLE
Model implementation diff from paper

### ISSUE THREAD

**Issue by Yitian-Li** [amirhfarzaneh/lsoftmax-pytorch#3]
thanks for your work，but maybe there is some problems in models.py.
In paper，conv_2 is 3 conv2d
![image](https://user-images.githubusercontent.com/48313312/65432162-50259880-de4d-11e9-8455-251e29c6eb5d.png)
while in your implementation there are 4 conv2d
 `        self.conv_2 = nn.Sequential(
            nn.Conv2d(64, 64, 3, padding=1),
            nn.PReLU(),
            nn.BatchNorm2d(64),
            nn.Conv2d(64, 64, 3, padding=1),
            nn.PReLU(),
            nn.BatchNorm2d(64),
            nn.Conv2d(64, 64, 3, padding=1),
            nn.PReLU(),
            nn.BatchNorm2d(64),
            nn.Conv2d(64, 64, 3, padding=1),
            nn.PReLU(),
            nn.BatchNorm2d(64),
            nn.MaxPool2d(2, 2)
        )`
